script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -104,7 +104,7 @@ I get the feeling that if I keep buttering her up like this,
 -- 死者は互いに食い合わないよう、犠牲者の数を一区間に集中させないよう、それぞれ活動ルートを定めているの。
 The Dead are each assigned their own non-overlapping routes,
  both to prevent them from trying and feed on each other
- and to ensure that the victims aren't all clustered around the same location.
+ and to ensure that the victims aren't all clustered around the same locations.
```

Comment:
Location singular makes sense here I think, if the dead patrol routes overlapped all the kills would be around the one area.

Reply:
Hmm I thought that making it locations was to more emphasize that there would be multiple locations where the killings may gather making the problem seem more wide scale. Do we have an idea on how big Roa's operation was?